Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1

```diff
@@ -480,3 +480,55 @@ function Sync-VMHostStorage {
 
     Get-VMHost $VMHostName | Get-VMHostStorage -RescanAllHba -RescanVMFS | Out-Null
 }
+
+<#
+    .SYNOPSIS
+     This function removes the specified static iSCSI configurations from all of Esxi Hosts in a cluster
+
+    .PARAMETER ClusterName
+     Cluster name
+
+    .PARAMETER iSCSIAddress
+     iSCSI target address. Multiple addresses can be seperated by ","
+
+    .EXAMPLE
+     Remove-VMHostStaticiSCSITargets -ClusterName "myCluster" -ISCSIAddress "192.168.1.10,192.168.1.11"
+
+    .INPUTS
+     vCenter cluster name and iSCSi target address
+
+    .OUTPUTS
+     None
+#>
+function Remove-VMHostStaticiSCSITargets {
```

Comment:
Could this be renamed to Remove-VMHostStaticIScsiTargets?

Reply:
Sure